test/core/util/tls_utils.h

```diff
@@ -43,6 +43,16 @@ PemKeyCertPairList MakeCertKeyPairs(absl::string_view private_key,
 
 std::string GetFileContents(const char* path);
 
+//  Checks that the string `private_key` is that of a key of supported type and
+//  in PKCS #8 syntax. Returns an OK status if so or a not-OK status otherwise.
+//  Logs supported but unexpected types.
```

Comment:
The log comment is not required. Please remove.

Reply:
Noted, thanks